Gotify.json

```
@@ -0,0 +1,39 @@
+{
+    "Gotify": {
+        "description": "A simple server for sending and receiving messages. <p>Based on the official Gotify x86 docker image: <a href='https://hub.docker.com/r/gotify/server' target='_blank'>https://hub.docker.com/r/gotify/server</a>.</p>",
+        "ui": {
+            "slug": ""
+        },
+        "website": "https://gotify.net/",
+        "version": "latest",
+        "containers": {
+            "gotify": {
+                "image": "gotify/server",
+                "launch_order": 1,
+                "ports": {
+                    "80": {
+                        "description": "Gotify server port. Suggested default: 80",
+                        "host_default": 80,
```

While setting this to `80` does work as Rockstor tries the next available port, it can be risky to leave it set at 80. In my case, for instance, Rockstor found the next one available to be 87 and while it does not break the Rock-On install, it does prevent access to Gotify's UI. Indeed, Firefox blocked access to it as port 87 is reserved for another protocol.
I'll file an issue to improve our Rockstor deals with this, but to circumvent this for the moment, I would recommend setting a `host_default` value to a high port number such as 8100. You can set it to whatever else you want in that "high number" space, though.
```suggestion
                        "host_default": 8100,
```